### PR TITLE
Add generic Superset guide

### PR DESCRIPTION
**Pull request by hemajv**
## Related Issues and Dependencies

#53 

## This introduces a breaking change

- [ ] Yes
- [X] No

## Description

Add a markdown guide for Superset

### CODE REVIEW

File: data-presentation/superset.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
+# Superset
+
+[Apache Superset](https://superset.apache.org/) is an open source modern data exploration and visualization platform. Superset can query data from any SQL-speaking data store or data engine. Some of the databses supported are [PostgreSQL](https://www.postgresql.org/), [MySQL](https://www.mysql.com/), [Trino](https://trino.io/) etc. Superset comes with an intuitive interface for visualizing datasets and creating interactive dashboards. It also has an in-built SQL lab to explore and test your queries.
```
Comment (pacospace) [inline data-presentation/superset.md:3]: `databases`

File: data-presentation/superset.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+## Getting Started with Superset
+
+**Step 1: Installing Superset**
+
+Superset can be installed on many different operating systems.  For a list of the minimum hardware and software requirements, as well as instructions on installing Superset, refer to [Install Superset](https://superset.apache.org/docs/installation/installing-superset-using-docker-compose).
+
+We have Superset deployed in one of the [Operate First](https://www.operate-first.cloud/) instances and it can be accessed [here](https://superset.operate-first.cloud/).
+The Operate First Superset primarily uses a Trino database, but also onboards other teams to connect their own databases. You can follow the instructions for accessing and creating dashboards [here](https://www.operate-first.cloud/users/support/docs/trino_superset_user_guide.md).
```
Comment (pacospace) [inline data-presentation/superset.md:12]: [Trino](https://trino.io/)

File: data-presentation/superset.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+**Step 2: Connecting to Databases**
+
+Superset requires a Python DB-API database driver and a SQLAlchemy dialect to be installed for each datastore you want to connect to.
+You can read more [here](https://superset.apache.org/docs/databases/dockeradddrivers) about how to install new database drivers into your Superset configuration.
+
+You’ll need to install the required packages for the database you want to use as your metadata database as well as the packages needed to connect to the databases you want to access through Superset. For example, if you wish to use Trino you will need to install the required package:
+
+```python
+pip install sqlalchemy-trino
+
```
Comment (pacospace) [inline data-presentation/superset.md:23]: many blank lines

File: data-presentation/superset.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+
+```
+For more information on configuring databases refer [here](https://superset.apache.org/docs/databases/installing-database-drivers).
+
+**Step 3: Adding Database to Superset**
+
+Once you have the database drivers installed and connected, you can login to Superset add the database from the UI by:
```
Comment (pacospace) [inline data-presentation/superset.md:30]: ...`and` add the database from the UI

File: data-presentation/superset.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+
+1. Clicking on `Data -> Databases -> + Database`
+2. Provide the database configuration
+
+<img src="../assets/images/superset-add-db.png" width="700">
+
+3. Test to see if the database connection is successful
```
Comment (pacospace) [inline data-presentation/superset.md:37]: what type of test?

**Review by pacospace — COMMENTED** [review]
Very nice!! 🚀 few mintor comments

File: data-presentation/superset.md (hunk 5 above)
Comment (hemajv) [reply data-presentation/superset.md:37]: oh basically just click on the Save button and it should just display a message if the connection was successful. let me rephrase it

**Review by pacospace — APPROVED** [review]
